From akpm@linux-foundation.org  Mon Nov 26 14:58:58 2007
From: Adrian Bunk <bunk@kernel.org>
Date: Wed, 21 Nov 2007 15:07:12 -0800
Subject: PCI: remove additional pci_scan_child_bus() prototype
To: greg@kroah.com
Cc: akpm@linux-foundation.org, bunk@kernel.org
Message-ID: <200711212307.lALN7CGE013912@imap1.linux-foundation.org>


From: Adrian Bunk <bunk@kernel.org>

There's already a prototype for pci_scan_child_bus() at the correct place in
pci.h, so there's no reason for an additional one.

Signed-off-by: Adrian Bunk <bunk@kernel.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/pci/probe.c |    2 --
 1 file changed, 2 deletions(-)

--- a/drivers/pci/probe.c
+++ b/drivers/pci/probe.c
@@ -487,8 +487,6 @@ static void pci_fixup_parent_subordinate
 	}
 }
 
-unsigned int pci_scan_child_bus(struct pci_bus *bus);
-
 /*
  * If it's a bridge, configure it and scan the bus behind it.
  * For CardBus bridges, we don't scan behind as the devices will
